prow/plugins/respond.go

```diff
@@ -27,6 +27,11 @@ const AboutThisBot = "Instructions for interacting with me using PR comments are
 
 // FormatResponse nicely formats a response to an issue comment.
 func FormatResponse(ic github.IssueComment, s string) string {
+	return FormatResponseRaw(ic.Body, ic.HTMLURL, ic.User.Login, s)
+}
+
+// FormatResponseRaw nicely formats a response for one does not have an issue comment
+func FormatResponseRaw(comment, commentURL, login, reply string) string {
```

Comment:
Note that in the details block it says "In response to this comment," but it may not be a comment. Change that to say something less specific like "In response to [this]" or something.